### PR TITLE
Maint: Show `Cmd + K` for macOS users

### PR DESCRIPTION
**Pull request by luckrnx09**

### CODE REVIEW

File: commafeed-client/src/components/sidebar/TreeSearch.tsx [hunk 1]
```diff
@@ -12,6 +12,10 @@ export interface TreeSearchProps {
     feeds: Subscription[]
 }
 
+function isMacOS() {
+    return navigator.platform.toUpperCase().includes("MAC")
```
Comment (luckrnx09) [inline commafeed-client/src/components/sidebar/TreeSearch.tsx:16]: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform

**Comment by Athou** [comment]
Thank you, that's a nice addition. Could you update the Keyboard Shortcuts help page in KeyboardShortcutsHelp.tsx too please?

Also, there seems to be [a helper in mantine to detect the user OS](https://mantine.dev/hooks/use-os/), I think it would be nice to use it since [it seems to be more robust](https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/hooks/src/use-os/use-os.ts).

**Comment by luckrnx09** [comment]
> Thank you, that's a nice addition. Could you update the Keyboard Shortcuts help page in KeyboardShortcutsHelp.tsx too please?
> 
> Also, there seems to be [a helper in mantine to detect the user OS](https://mantine.dev/hooks/use-os/), I think it would be nice to use it since [it seems to be more robust](https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/hooks/src/use-os/use-os.ts).

Thanks @Athou , mantine is great! I used the `useOs` hook to determine os type in this PR.

**Comment by Athou** [comment]
Awesome, thank you!

**Pull request merged**